pkg/jsonmessage/jsonmessage.go

```diff
@@ -189,13 +189,9 @@ func DisplayJSONMessagesStream(in io.Reader, out io.Writer, terminalFd uintptr,
 				if isTerminal {
 					fmt.Fprintf(out, "\n")
 				}
-			} else {
-				diff = len(ids) - line
 			}
-			if isTerminal {
-				// NOTE: this appears to be necessary even if
-				// diff == 0.
-				// <ESC>[{diff}A = move cursor up diff rows
+			diff = len(ids) - line
```

Comment:
I think it is a good idea to add a comment saying that if `ok` is `false` then line is `0`.